src/declarations/stencil-public-docs.ts

```diff
@@ -19,6 +19,10 @@ export interface JsonDocsComponent {
   readme: string;
   docs: string;
   docsTags: JsonDocsTag[];
+  /**
+   * The comment found at in a class-level JSDoc for a Stencil component.
```

Comment:
maybe

```suggestion
   * The text from the class-level JSDoc for a Stencil component, if present.
```

Reply:
Bah! How did I miss this? I'll put a tiny followup PR for this. Sorry about that!